smarts/core/utils/file.py

```diff
@@ -94,3 +95,17 @@ def smarts_log_dir() -> str:
 
 def make_dir_in_smarts_log_dir(dir):
     return os.path.join(smarts_log_dir(), dir)
+
+
+@contextmanager
+def supress_pkg_resources():
+    import sys
+
+    import pkg_resources
+
+    from smarts.core.utils.invalid import raise_import_error
+
+    pkg_res = sys.modules["pkg_resources"]
+    sys.modules["pkg_resources"] = property(raise_import_error)
```

Comment:
If you are wondering what this does, when the module is attempted to be accessed it instead calls the method which raises an import error. This would normally indicate that the dependency the module is a part of does not exist.